src/main/java/dev/isxander/controlify/ingame/InGameInputHandler.java

```diff
@@ -313,7 +316,11 @@ protected void handleRegularLook(InputComponent input, Vector2f impulse, boolean
     protected void handleGyroLook(GyroComponent gyro, Vector2f impulse, boolean aiming) {
         GyroComponent.Config config = gyro.confObj();
 
-        if (config.requiresButton && (!ControlifyBindings.GYRO_BUTTON.on(controller).digitalNow() && !aiming)) {
+        if (config.requiresButton.equals(GyroButtonMode.ON) && (!ControlifyBindings.GYRO_BUTTON.on(controller).digitalNow() && !aiming)) {
```

Comment:
maybe separate all these usages of `ControlifyBindings.GYRO_BUTTON.on(controller)` into a local called gyroButton to make the code a little cleaner!

Reply:
made the changes